src/python/grpcio/grpc/_channel.py

```diff
@@ -24,6 +24,8 @@
 from grpc._cython import cygrpc
 from grpc.framework.foundation import callable_util
 
+_LOGGER = logging.getLogger(__name__)
```

Comment:
The logger name should be lower case, Upper case should only be used when naming a constant, and the logger is not a constant.

Reply:
Why is the logger not a constant? Where is it reassigned or mutated?

Reply:
IMO, a constant is a simple immutable object i.e int, string, date, etc and not a complex object such as logger. The logger instance's properties and variable can be changed in any time when configuring python's logging. I'm not expecting this kind of behavior from a constant.

Reply:
@davidt99: while respecting that other codebases have other conventions and norms, the practice in gRPC Python is to use `_CONSTANT_STYLE_NAMING` for objects and values that _are maintained as constants_, not just for values like ints and strings that because they are immutable are necessarily constant. In the cases in this pull request, the styling as `_LOGGER` indicates that although the object supports having properties and variables changed some time after its construction, the code that uses the object does not actually change anything about it. Thanks much for reviewing!

@ghostwriternr: squash commits and I think this will be ready to be integrated?